lib/ansible/modules/network/aci/aci_maintenance_policy.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: aci_maintenance_policy
+short_description: Manage firmware maintenance policies
+version_added: '2.8'
+description:
+- Manage maintenance policies that defines behavior during an ACI upgrade.
+options:
+  name:
+    description:
+    - The name for the maintenance policy.
+    required: true
+    aliases: [ maintenance_policy ]
+  runmode:
+    description:
+    - Whether the system pauses on error or just continues through it.
+    - The default is C(yes), i.e. to pause.
+    type: bool
+    default: yes
+  graceful:
+    description:
+    - Whether the system will bring down the nodes gracefully during an upgrade, which reduces traffic lost.
+    type: bool
+  scheduler:
+    description:
+    - The name of scheduler that is applied to the policy.
+    type: str
+    required: true
+  upgrade:
```

Comment:
To me it is unclear why it is called `upgrade` here, but the description mentions `adminst`.